packages/components/src/components/CollapsibleBox/types.tsx

```diff
@@ -0,0 +1,10 @@
+import { uiSizes } from '../../config/types';
+
+export const paddingTypes = ['none', 'normal', 'large'] as const;
+export type PaddingType = (typeof paddingTypes)[number];
+
+export const fillTypes = ['none', 'default'] as const;
+export type FillType = (typeof fillTypes)[number];
+
+export const headingSizes = uiSizes.filter(size => size !== 'tiny');
```

Comment:
Why not define it explicitely? It can break in the future.

Reply:
Fixed in the latest update